pkg/webrtc/ice.go

```diff
@@ -0,0 +1,62 @@
+package webrtc
+
+import (
+	"strings"
+
+	"github.com/giongto35/cloud-game/v2/pkg/config/webrtc"
+)
+
+type Replacement struct {
+	From string
+	To   string
+}
+
+func NewIceServer(url string) webrtc.IceServer {
+	return webrtc.IceServer{
+		Url: url,
+	}
+}
+
+func NewIceServerCredentials(url string, user string, credential string) webrtc.IceServer {
+	return webrtc.IceServer{
+		Url:        url,
+		Username:   user,
+		Credential: credential,
+	}
+}
+
+func ToJson(iceServers []webrtc.IceServer, replacements ...Replacement) string {
+	var sb strings.Builder
+
+	n := len(replacements)
+	serversN := len(iceServers)
```

Comment:
Look like this can be rewritten better :/ 
We can use strings.Join 2 times, one to build a formatted version of each iceServer, and then join all iceServers together

Reply:
This' a superfast version (: We can just json.Marshal(iceServers) + loop replacements (:

Reply:
Need to test what's faster. Essentially, strings.join on iceServers will do exactly the same logic. I should rewrite strings.Join on iceServer params (: and bench.

Reply:
```
goos: windows
goarch: amd64
pkg: github.com/giongto35/cloud-game/v2/pkg/webrtc
BenchmarkIces
BenchmarkIces/toJsonOriginal
BenchmarkIces/toJsonOriginal-4         	  989529	      1171 ns/op	     520 B/op	       9 allocs/op
BenchmarkIces/toJsonJoins
BenchmarkIces/toJsonJoins-4            	  480134	      2430 ns/op	    1120 B/op	      27 allocs/op
BenchmarkIces/toJsonMarshal
BenchmarkIces/toJsonMarshal-4          	  599916	      1933 ns/op	     864 B/op	       5 allocs/op
PASS
```
As expected, my implementation without joins is the fastest (original is ~1800 ns/op), joins is the slowest, and simple marshaling 2nd (: